wp-content/themes/missilethreat/archive.php

```diff
@@ -43,26 +45,137 @@
+
+      <?php
+      $args = array(
+        'post_type' => 'actors',
+        'numberposts' => -1,
+        'orderby' => 'meta_value',
+        'order' => 'ASC',
+        'meta_key' => '_actors_archive_name'
+      );
+      $actors = get_posts( $args ); ?>
+
+      <h2 class="actors__header actors__header--lower">Countries</h2>
+        <div class="actors__group actors__group-countries">
+
+          <?php
+          foreach ( $actors as $post) {
+            setup_postdata( $post );
+            $actor_icon = get_field('country_icon', $post->ID);
+            $archive_name = get_the_title();
+            $replacement_archive_name = get_post_meta( $post->ID, '_actors_archive_name', true );
+
+            if ( $replacement_archive_name ) {
+              $archive_name = $replacement_archive_name;
+            }
+
+            if ( $post->post_name === 'hezbollahs-rocket-arsenal' ) {
+              continue;
+            }
+
+            ?>
+
+            <div class="actors__container">
+              <a href="<?php echo esc_url( get_permalink() ); ?>" class="actors__link">
+                <img src="<?php echo esc_url($actor_icon['url']); ?>" alt="<?php echo esc_attr($actor_icon['alt']); ?>" class="actors__icon">
+                <?php echo $archive_name; ?>
+              </a>
+            </div>
+          <?php }  ?>
+      </div><!-- .actors__countries -->
+
+
+      <h2 class="actors__header">Sub-state Actors</h2>
+        <div class="actors__group actors__group-sub-state">
+
+          <?php
+          foreach ( $actors as $post) {
+            setup_postdata( $post );
+            $actor_icon = get_field('country_icon', $post->ID);
+            $archive_name = get_the_title();
+            $replacement_archive_name = get_post_meta( $post->ID, '_actors_archive_name', true );
+
+            if ( $replacement_archive_name ) {
+              $archive_name = $replacement_archive_name;
+            }
+
+            if ( $post->post_name !== 'hezbollahs-rocket-arsenal' ) {
+              continue;
+            }
+
+            ?>
+
+            <div class="actors__container">
+              <a href="<?php echo esc_url( get_permalink() ); ?>" class="actors__link">
+                <img src="<?php echo esc_url($actor_icon['url']); ?>" alt="<?php echo esc_attr($actor_icon['alt']); ?>" class="actors__icon">
+                <?php echo $archive_name; ?>
+              </a>
+            </div>
+          <?php }  ?>
+
+      </div><!-- .actors__sub-state -->
+    </section>
+    <?php }
+
+  elseif ( is_post_type_archive('defsys') ) { ?>
+    <section class="defsys">
+
+      <h2 class="actors__header">U.S. Defense Systems</h2>
```

Comment:
See my notes above about making this text editable via the dashboard. I'd also update the descriptions to be paragraphs and the elements to be list items.